src/components/dropdown/dropdown.ts

```diff
@@ -246,7 +264,9 @@ export class MDropdown extends BaseDropdown implements MDropdownInterface {
         });
         this.internalItems = items;
         this.internalNavigationItems = navigation;
-        this.$refs.popup.update();
+        if (this.as<MediaQueries>().isMqMinS) {
```

Comment:
J'aimerais vraiment pas qu'on utilise des refs partout.  En gros, ici on demande à la popup de s'updater pcq le texte vient de s'updateré

1-est-ce vraiment nécéssaire?
2-pourquoi est-ce qu'on le fait juste si isMqMinS?
3-faudra éventuellement se mettre à utiliser le hook updated de vue ou des mutationobserver (le hook updated est large, callé souivent.  On peut mettre des mutationobserver spécifique) afin de ne plus avoir à caller les update à la main.  La popup devrait s'avoir quand s'updater :-).

Je suis conscient que tu as juste déplacé le call de update, mais je pose quand même la question à savoir pourquoi on a ajouté la condition sur ismqmins

Reply:
**Réponse question 1**
`this.$refs.popup.update()` est utilisé pour updater la position du popup. Elle est donc nécessaire

**Réponse question 2**
Le `isMqMinS` est utilisé, car la méthode `update` du composant **m-popup** ne fonctionne qu'en mode desktop. 

**Réponse question 3**
Oui, je crois que ce serait bien d'utiliser un mutation observer dans le composant **m-popper** au lieu d'appeler la méthode update avec des $refs.

Je faire ces modifications dans ce PR.